lib/kubeclient.rb

```diff
@@ -1,6 +1,7 @@
 require 'json'
 require 'rest-client'
 
+require 'kubeclient/aws_eks_credentials'
```

Comment:
This will require adding it as a dependency for everyone, no?

Reply:
Aah wait, it's our helper not external library.  I need to look at this when I'm less tired :-)

Reply:
LGTM :+1:
Our README says one can use `Kubeclient::AmazonEksCredentials` doesn't say one would need to require it (which would be strange, `require 'kubeclient` is enough for everything else).
aws_eks_credentials.rb only requires `aws-sigv4` when actually used.